workflows/sarek/main.nf

```diff
@@ -20,6 +20,13 @@ include { CHANNEL_VARIANT_CALLING_CREATE_CSV          } from '../../subworkflows
 include { BAM_CONVERT_SAMTOOLS as CONVERT_FASTQ_INPUT } from '../../subworkflows/local/bam_convert_samtools/main'
 include { BAM_CONVERT_SAMTOOLS as CONVERT_FASTQ_UMI   } from '../../subworkflows/local/bam_convert_samtools/main'
 
+// Convert fastq.gz.spring files to fastq.gz files
+include { SPRING_DECOMPRESS as SPRING_DECOMPRESS_ONE_OF_TWO } from '../../modules/nf-core/spring/decompress/main'
+include { SPRING_DECOMPRESS as SPRING_DECOMPRESS_TWO_OF_TWO } from '../../modules/nf-core/spring/decompress/main'
+include { SPRING_DECOMPRESS as SPRING_DECOMPRESS_ONE_OF_ONE } from '../../modules/nf-core/spring/decompress/main'
```

Comment:
why not, SPRING_DECOMPRESS_ONE_OF_ONE -> SPRING_DECOMPRESS_DUO
SPRING_DECOMPRESS_ONE_OF_TWO -> SPRING_DECOMPRESS_FIRST_PAIR
SPRING_DECOMPRESS_TWO_OF_TWO -> SPRING_DECOMPRESS_SECOND_PAIR

Reply:
Hmmm... 🤔 I don't think I could tell what the purpose of the module "instance" from the names you proposed. (Of course, "my" names weren't any better.)

Basically, I prefer long and clumsy names to ambiguous names. I would prefer some crazy long names like:

SPRING_DECOMPRESS_DUO -> SPRING_DECOMPRESS_TO_FQ_PAIR
SPRING_DECOMPRESS_ONE_OF_TWO -> SPRING_DECOMPRESS_TO_FIRST_FQ_OF_FQ_PAIR
SPRING_DECOMPRESS_TWO_OF_TWO -> SPRING_DECOMPRESS_TO_SECOND_FQ_OF_FQ_PAIR

Anyways, we can go with your suggestion as it really doesn't matter much to me.

Reply:
ok for `SPRING_DECOMPRESS_TO_FQ_PAIR`
what about `SPRING_DECOMPRESS_TO_FIRST_FQ_OF_FQ_PAIR` -> `SPRING_DECOMPRESS_TO_R1_FQ_PAIR`